Issue36 sub project spin off (#49)
confidential-computing/governance: project-progression-policy.md

* The project will need to make these changes in order to progress further.
* Projects are accepted or progress from level to level via a majority vote of TAC representatives present if quorum is reached or a majority of all TAC representatives if voting by email, unless a project is being accepted at or moved to either the Graduation or Emeritus stage, in which a vote of two-thirds of all TAC representatives is required.
* The proposal document will be finalized and a project charter prepared. This charter document must be included in the project's main repository.
* The TAC will determine the appropriate initial stage for the project. The project can apply for a different stage via the review process.

### Spin-off Projects ###

> Projects can create multiple subordinate projects that are governed under the structure agreed and accepted by the TAC.

Reviewer: ```suggestion
Projects can create multiple subordinate projects that are governed under the structure agreed on and accepted by the TAC.
```

> A project may want to spin-out a new project becuase the original scope, governance or participants are distinctly different to the original agreed project charter, and not merely an organization of code or minor expansion of scope.

Reviewer: ```suggestion
A project may want to spin out a new project because the original scope, governance or participants are distinctly different from the original agreed project charter, and not merely an organization of code or minor expansion of scope.
```

In this case a new project acceptance proposal, as defined above, should be submitted to the TAC.

## III. Stages - Definitions & Expectations
Every Consortium project has an associated maturity level. Proposed projects should state their preferred maturity level.  There are four maturity levels under this Project Progression Policy:

* Sandbox - projects looking for a foundation with minimal initial resource requirements;

> * Incubation - growing projects that targeting particular growth metrics and looking for resources from the Consortium;

Reviewer 2: that _are_ targeting...

> * Incubation - growing projects that targeting particular growth metrics and looking for resources from the Consortium;

Author: ```suggestion
* Incubation - growing projects that are targeting particular growth metrics and looking for resources from the Consortium;
```
